### ISSUE TITLE
Visual Studio 2015, Resharper 2016.1 & 1.4.0 not working

### ISSUE THREAD

**Issue by stuartleaver** [machine/machine.specifications.runner.resharper#38]
I am running Visual Studio 2015 and have just upgraded to ReSharper 2016.1 to allow me to install version 1.4.0 of the Machine.Specifications.Runner.Resharper. Version details are:

> JetBraines ReSharper Ultimate 2016.1.1
Build 105.0.201.605.0504.101434

> Microsoft Visual Studio Professional 2015
Visual Studio 14.0.23107.0 

> Machine.Specifications.Runner.Resharper for ReSharper 2016.1 1.4.0

The ReSharper test runner does discover the MSpec tests but when run, the following error is encountered:

> Unit Test Runner failed to run tests
Constructor on type 'Machine.Specification.Runner.Impl.DefaultRunner' not found

I do have this VSIX installed in Visual Studio but I wouldn't have thought that would make any difference - https://visualstudiogallery.msdn.microsoft.com/f8d80e41-0a85-4b04-aa41-0bc59dc81a5a/file/168698/3/Machine.VSTestAdapter.vsix

I'm assuming the above setup of versions should allow me to run MSpec tests through ReSharper 2016.1 as the version is also in the extension name. So are there any ideas on why I am getting the above error please?

**Comment by Schaeri** [comment]
I have never tested the runner with R# 2016.1.1 only 2016.1. Maybe we have a problem with the new version. I will check if the runner works with 2016.1.1 tomorrow. Do you also have the same problem when using R# 2016.1?

**Comment by stuartleaver** [comment]
I haven't tested it with 2016.1. The same error did occur with R# 10 and the extension manager only found version 1.3.0 of the runner. Hence upgrading to R# 2016.1.1 which I did through the builtin update checker when I noticed 1.4.0 had been released with support.

I did read else where it being mentioned that the version of the actual Machine.Specification framework could cause the problem. Are you aware if that could be a cause?

It would be good if you could test tomorrow and I'll also see if I can find a download of 2016.1. It would be good to get it working though because the VSIX I linked to is far from ideal and seems to be a bit hit and miss when discovering tests. For example, in a solution of 2800+ tests, it only found around 80 at one point today.

**Comment by stuartleaver** [comment]
As an update, I have updated the Machine.Specification Nuget package to 0.9.0 and the Runner seems to be working now. Not sure why it didn't go to 0.9.3 but when I upgraded Machine.Specification.Should the dependencies just went to 0.9.0 but thats' neither here nor there in the context of this issue.

Therefore, I would say that the issue was due to using an old Machine.Specification version rather than the actual Runner.
